shuttle/templates/schedules/load_schedule.html

```diff
@@ -1,61 +1,29 @@
 {% block body_content %}
 
-    <div class="column">
-        <div class="card">
-            <div class="card-body log-body">
-                <table class="table">
-                    <tbody>
-                        {% for i in range(schedule|length) %}
-                            {% if i % 2 == 0 %}
-                                <tr class="even-row">
-                                    <td>
-                                        <h6 class="schedule-location">{{ schedule[i][0] }}</h6>
-                                    </td>
-                                </tr>
-                            {% else %}
-                                <tr class="even-odd">
-                                    <td>
-                                        <h6 class="schedule-location">{{ schedule[i][0] }}</h6>
-                                    </td>
-                                </tr>
-                            {% endif %}
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-        </div>
-    </div>
-
-    <div class="card scrolling-wrapper">
-        <div class="card-body log-body">
-            <table class="table table-hover">
-                <tbody>
-                {% for i in range(schedule|length) %}
-                    {% if i % 2 == 0 %}
-                        <tr class="even-row">
-                            {% for j in range(schedule[i]|length) %}
-                                <td>
-                                    {% if j != 0 %}
-                                        <h6 class="schedule-time">{{ schedule[i][j] }}</h6>
-                                    {% endif %}
-                                </td>
-                            {% endfor %}
-                        </tr>
+<div class="card scrolling-wrapper">
+    <div class="card-body log-body">
+        <table class="table table-hover">
+            <tbody>
+            {% for i in range(schedule|length) %}
+                <tr>
+                    {% for j in range(schedule[i]|length) %}
+                        {% if j % 2 == 0 %}
+                            <td class="even-column">
+                                {% else %}
+                            <td>
+                        {% endif %}
+                    {% if i == 0 %}
+                        <p class="schedule-time"><b>{{ schedule[i][j] }}</b></p>
```

Comment:
Could you make the space between the columns smaller. 

Right now there is a lot of extra space between columns that makes the table excessively large imo.